functionaltests/src/test/java/util/constant/WebDriverCommon.java

```diff
@@ -5,7 +5,7 @@
  */
 public final class WebDriverCommon {
 
-    public static final String CSS_SELECTOR_SUBMIT = "button[type=submit]";
+    public static final String XPATH_SUBMIT = "//button[@type='submit']";
```

Comment:
How is this different from the previous solution? If you have 2 submit buttons, it's still not identified properly, right?